LEGO1/mxdsfile.cpp

```diff
@@ -1,6 +1,130 @@
 #include "mxdsfile.h"
 
+#include <stdio.h>
+
+#define SI_MAJOR_VERSION 2
+#define SI_MINOR_VERSION 2
+
+#define FCC_OMNI 0x494e4d4f
+#define FCC_MxHd 0x6448784d
+#define FCC_MxOf 0x664f784d
```

Comment:
For another project, I created `FOURCC` macros for this:
```c
#define FOURCC(a, b, c, d) (((a) << 0) | ((b) << 8) | ((c) << 16) | ((d) << 24))
```

This allows you to redefine these macros as:
```c
#define FCC_OMNI FOURCC('O', 'M', 'N', 'I')
#define FCC_MxHd FOURCC('M', 'x', 'H', 'd')
#define FCC_MxOf FOURCC('M', 'x', 'O', 'f')
```